web/src/realm_logo.ts

```diff
@@ -87,12 +88,23 @@ export function render(): void {
         $("#realm-night-logo-upload-widget .image-block").attr("src", realm.realm_night_logo_url);
     }
 
+    const $realm_logo = $<HTMLImageElement>("#realm-navbar-wide-logo");
     if (settings_data.using_dark_theme() && realm.realm_night_logo_source !== "D") {
-        $("#realm-navbar-wide-logo").attr("src", realm.realm_night_logo_url);
+        $realm_logo.attr("src", realm.realm_night_logo_url);
     } else {
-        $("#realm-navbar-wide-logo").attr("src", realm.realm_logo_url);
+        $realm_logo.attr("src", realm.realm_logo_url);
     }
 
+    $realm_logo.on("load", () => {
+        const logo_width = $realm_logo.width();
+        if (logo_width) {
+            $("html").css(
+                "--realm-logo-current-width",
+                logo_width / user_settings.web_font_size_px + "em",
+            );
+        }
+    });
```

Comment:
https://api.jquery.com/load-event/ doesn't make me feel very confident that this code will run consistently.

In particular, if the image is already cached, does this get called at all?

Reply:
Yeah, this is working well for even cached images. I tested on chrome and firefox.